website/docs/getting-started/basics.md

```diff
@@ -8,15 +8,34 @@ odo abstracts Kubernetes concepts into developer focussed terminology. In this d
 
 ### Concepts
 
-#### Component
+* Component
+  A component is a microservice. Each component can be run and deployed separately. Examples of components: Warehouse API Backend, Inventory API, Web Frontend, Payment Backend
+  > A component in odo is an equivalent of a deployment or deploymentconfig in Kubernetes. Creating a new component in odo will create a deployment in Kubernetes or for git and binary based component creation, it will create deploymentconfig.
 * Application
+  An application consists of multiple components which may span over multiple projects, and work individually to build the entire application. Examples of applications: e-Shop, Hotel Reservation System, Online Booking
```

Comment:
> An application consists of multiple components which may span over multiple projects

This is not true for odo. odo doesn't cover things across multiple namespaces.

Reply:
I meant that an application can be accessed over multiple namespaces. But I see how it might have come off to mean that a user can access the same component over multiple namespaces, is that what you meant?

Anyway, I'll try to fix that.

Reply:
What I understood is that the components can be distributed over different namespaces. Which is not true for odo. At least not yet.